### PR TITLE
Added orderBy option to getProjectUpdate endpoint

### PR DESCRIPTION
**Pull request by aminlatifi**

### CODE REVIEW

File: src/resolvers/projectResolver.ts [hunk 1]
```diff
@@ -1036,9 +1036,17 @@ export class ProjectResolver {
...
     @Arg('take', type => Int, { defaultValue: 10 }) take: number,
     @Arg('connectedWalletUserId', type => Int, { nullable: true })
+    @Arg('orderBy', type => OrderBy, {
+      defaultValue: {
+        field: OrderField.CreationAt,
+        direction: OrderDirection.DESC,
+      },
+    })
+    orderBy: OrderBy,
     connectedWalletUserId: number,
```
Comment (CarlosQ96) [inline src/resolvers/projectResolver.ts]: Move this variable to it's corresponding Argument.

Comment (CarlosQ96) [reply src/resolvers/projectResolver.ts]:
```javascript
@Arg('connectedWalletUserId', type => Int, { nullable: true })
connectedWalletUserId: number,
```

File: src/resolvers/projectResolver.ts [hunk 2]
```diff
@@ -1047,6 +1055,7 @@ export class ProjectResolver {
           projectId,
         },
       )
+      .orderBy(field, direction)
```
Comment (CarlosQ96) [inline src/resolvers/projectResolver.ts]: For some reason, typeorm not recognizing the column. 
Usually we interpolate the entity like this.
```javascript
.orderBy(`projectUpdate.${field}`, direction)
```

**Review by CarlosQ96 — CHANGES_REQUESTED** [review]

File: src/resolvers/projectResolver.ts (hunk 2 above)
Comment (aminlatifi) [reply src/resolvers/projectResolver.ts]: Nice, good catch

**Review by CarlosQ96 — APPROVED** [review]
LGTM thanks man!

**Pull request merged**